### DISCUSSION TITLE
IS THARE A WAY TO USE NET 64 WITH THIS

### DISCUSSION THREAD

**Discussion by chasonic** [gonetz/GLideN64#2546] [Q&A]
IS THARE A WAY TO USE NET 64 WITH THIS

**Comment by oddMLan** [comment]
NO NET64 USES A CUSTOM VERSION OF PROJECT64 WITH EXPANDED MEMORY AND A MODIFIED GLIDE64 SO THIS DOESNT WORK WITH IT
